mkosi/__init__.py

```diff
@@ -2464,7 +2464,10 @@ def add_apt_auxiliary_repos(state: MkosiState, repos: Set[str]) -> None:
 
     # Security updates repos are never mirrored
     if state.config.distribution == Distribution.ubuntu:
-        security = f"deb http://security.ubuntu.com/ubuntu/ {state.config.release}-security {' '.join(repos)}"
+        if state.config.architecture == "aarch64":
```

Comment:
ports is used for other architectures I think? armhf, ppc64el, riscv64, s390x http://ports.ubuntu.com/dists/jammy/

Reply:
@marex Can you add these architectures as well here?